docs/types.rst

```diff
@@ -479,7 +439,7 @@ Here you can find a list of all types and default values:
 .. _type_conversions:
 
 Type Conversions
```

Comment:
This table uses older [constant names](https://vyper.readthedocs.io/en/release-0.2.0/constants-and-vars.html#built-in-constants), and also doesn't display all on one page